### DISCUSSION TITLE
project의 진행사항 파악이 쉽도록 projects의 kanban을 정리하는건 어떨까요?

### DISCUSSION THREAD

**Discussion by wooogi123** [buttercrab/wtfpython-ko#35] [Ideas]
projects의 translate kanban을

| TODO | WIP | DONE |
|---|:---:|---:|
| 번역필요 | 번역중 | 번역완료 |

같은 느낌으로 step을 분류해서 정리하는것은 어떨까요?
card를 예제별로 분류한다면 한눈에 파악이 보다 쉬울듯합니다.

**Comment by buttercrab** [comment]
좋은 생각인것 같습니다.
내일 만들어 보도록 할게요

**Reply by wooogi123** [reply]
제가 지금 한번 정리해보겠습니다. 내일 보고 리뷰해주시겠어요? 😃

**Reply by buttercrab** [reply]
넵 👍

**Comment by buttercrab** [comment]
@wooogi123 일일이 다 만드신 건가요?

**Reply by wooogi123** [reply]
네 지라처럼 ticket -> issue가 되는줄 알았는데 issue를 아예 티켓으로 사용하는 구조여서 정리할겸 한번 쭉 훑었습니다

**Reply by buttercrab** [reply]
대단하십니다 👍
